python/setup.py

```diff
@@ -237,8 +237,8 @@ def get_packages(self):
         ],
         "default": [
             # If adding dependencies necessary to launch the dashboard api server,
-            # please add it to python/ray/dashboard/optional_deps.py as well.
```

Comment:
could you keep this `python/ray` line? `dashboard` has been moved.

Reply:
whoops, must've been a merge error.